src/pages/index.js

```diff
@@ -49,59 +32,168 @@ const popupCardForm = document.forms.newCardForm;
 
 const btnEditProfile = document.querySelector('.profile__btn_type_edit');
 const btnNewCard = document.querySelector('.profile__btn_type_add');
+const btnEditProfileImage = document.querySelector('.profile-photo-edit__edit-btn');
 
 const containerSelector = document.querySelector('.cards');
 
 const popupProfileForm = document.forms.profileForm;
 const popupPicture = document.querySelector('.popup_picture');
+const popupConfirmDelete = document.querySelector('.popup_confirm-delete');
+const popupEditProfileImage = document.querySelector('.popup_edit-profile-image');
+const popupProfileImageForm = document.forms.profileEditImageForm;
 
-const cardList = new Section(
-  {
-    data: initialCards,
-    renderer: createCard,
-  },
-  containerSelector // .cards
-);
-
-cardList.renderItems();
+let cardList;
 
 const profileFormValidator = new FormValidator(options, popupProfileForm);
 profileFormValidator.enableValidation();
 
 const newCardFormValidator = new FormValidator(options, popupCardForm);
 newCardFormValidator.enableValidation();
 
-function createCard(cardData) {
-  const card = new Card(cardData, '.template-card', handlePicturePopup);
-  const cardElement = card.getCardElement();
-  cardList.addItem(cardElement);
-}
+const ProfileImageFormValidator = new FormValidator(options, popupProfileImageForm);
+ProfileImageFormValidator.enableValidation();
 
 const popupImage = new PopupWithImage(popupPicture);
 
 function handlePicturePopup(name, link) {
   popupImage.open(name, link);
 }
 
-const popupNewCard = new PopupWithForm(newCard, (data) => {
-  createCard(data);
-  popupNewCard.reset();
-  newCardFormValidator.disableSubmitButton();
-});
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    '.template-card',
+    handlePicturePopup,
+    () => {
+      popupDelete.setAction(() => {
+        api
+          .deleteCard(cardData._id)
+          .then(() => {
+            cardElement.remove();
```

Comment:
Needs correction: You should add a method `remove` to the Card class and use it here `card.remove()`. it's not good to remove the card element directly since the Card class is responsible for it.